### ISSUE TITLE
Something funky with Paw theme in 1.1.31

### ISSUE THREAD

**Issue by ekgermann** [Cacti/cacti#1220]
I've update to 1.1.31 tonight and the Paw theme seems to be broken in the header.

Essentially, only the background shows up, with no icons or text.  Therefore, you can't navigate to or logout of anything

It looks like this:

![image](https://user-images.githubusercontent.com/9213892/35030756-18d376a8-fb15-11e7-9437-670ef31d5e16.png)

Paper Plane seems to do the same thing.  After dinking around for a while, I did manage to hack it and get a different theme in place and they work fine.

I cleared the cache in all the browsers and it's reproducible (at least for me) in:

Chrome 63.0.3239.132
FF Quantum 57.0.4
Opera 50.0.2762.58
Vivaldi 1.14.1064.3
Safari 11.0.2 (13604.4.7.1.6)

I DID clear all the caches as pointed out in the upgrade docs.

FWIW, it seems like with the cache cleared it takes forever to load the initial page when the theme is broken.  Not sure if it's trying to pull in a resource and blowing up on render because it isn't there or what.

**Comment by netniV** [comment]
Yes, I had just noticed this myself.  The same issue is also present on Paper-Plane

**Comment by netniV** [comment]
And at line 1644 of include/themes/paw/main.css is a .maintabs nav with a display: none

This needs changing to display:block for the top bar to reappear.  But I am going to check against a recent commit that purportedly 'fixed' an issue with Paw to see if its related.

**Comment by netniV** [comment]
It would appear to be part of the styling changes made to accommodate the new button layouts that were introduced.  The specific commit that introduced that element was https://github.com/Cacti/cacti/commit/a646bc194b3bbf550a4b41f26f534bcab063b40e though it may be that it only broke with further changes later.

**Comment by netniV** [comment]
This issue can now be closed.

**Comment by cigamit** [comment]
The source of the issue is in the GUI API.  I noticed it yesterday evening.  Submitting the fix shortly.

**Comment by cigamit** [comment]
Please test out the updated lib/html.php.

**Comment by netniV** [comment]
Should I revert the block change too?

**Comment by cigamit** [comment]
Test it.  Might not be an issue.  If it's not an issue, don't worry about it.

**Comment by netniV** [comment]
OK, You put it in when you were fixing some of the theme stuff, so wondered if it was to do with any of the buttons etc.

**Comment by netniV** [comment]
Well, if I switch the block back to none it fails even with the updated html.php.

**Comment by netniV** [comment]
Also, I now seem to have a black bar across the bottom of the screen on 'Paw'

**Comment by netniV** [comment]
Yeah, I think https://github.com/Cacti/cacti/commit/e1c8a81811da9e313fab1aca0ffe19ca3438eb25 may have broken Tree.php again :)

**Comment by cigamit** [comment]
Then check the CSS history, as this was the original div layout.  So, it should be rending fine now.  It does on my systems

**Comment by netniV** [comment]
I can't seen an CSS issue to this.  In chrome it was rendering the layout incorrectly:

Button in wrong location:
![image](https://user-images.githubusercontent.com/9052188/35103872-127a69d8-fc5f-11e7-90d5-376a8b4c1d6a.png)

Once I reverted your commit, it works for me (not sure why that causes the other two issues).  But in reality, the real fix would be to remove the </div> you added to each element, and put it with the single </div> at the end of the if block.  The reason the above happens on the screenshot is that you are applying a </div> for every button we added to the Trees title bar which is incorrect (unless you planned to have a <div> too but didn't add it)
